pyemu/utils/helpers.py

```diff
@@ -3801,16 +3828,25 @@ def get_current_prop(_cur_thresh):
         d = a + ((b - a) / gr)
         numiter += 1
 
-    #print(a,b,c,d)
     thresh = (a+b) / 2.0
     prop = get_current_prop(thresh)
-    #print(thresh,prop)
     farr = np.zeros_like(tarr) - 1
     farr[tarr>thresh] = tfill[1]
     farr[tarr<=thresh] = tfill[0]
+    tarr[tarr>thresh] = tcat[0]
+    tarr[tarr <= thresh] = tcat[1]
+
     if iarr is not None:
         farr[iarr==0] = -1.0e+30
+        tarr[iarr == 0] = -1.0e+30
+    df.loc[tcat[0],"threshold"] = thresh
+    df.loc[tcat[1], "threshold"] = 1.0 - thresh
+    df.loc[tcat[0], "proportion"] = prop
+    df.loc[tcat[1], "proportion"] = 1.0 - prop
+    df.to_csv(csv_file.replace(".csv","_results.csv"))
     np.savetxt(orgarr_file,farr,fmt="%15.6E")
+    np.savetxt(tarr_file, tarr, fmt="%15.6E")
```

Comment:
ditto with the 'fmt' precision after #498, if great precision is required at some stage